experimental/dds/tree2/src/test/feature-libraries/editable-tree-2/editableTree.identifier.spec.ts

```diff
@@ -75,8 +75,9 @@ describe("editable-tree: node keys", () => {
 	it("can read local node keys", () => {
 		const { view, parentKey, childAKey, childBKey } = initializeView();
 		const parentNode = view.content;
-		const childA = parentNode.children.at(0);
-		const childB = parentNode.children.at(1);
+		// We know how the test tree looks and thus that these accesses won't return undefined.
+		const childA = parentNode.children.at(0)!; // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

Comment:
Nit: we should probably just add `assert(foo !== undefined`s here, rather than the non-null assertions.